mpas_analysis/ocean/streamfunction_moc.py

```diff
@@ -338,6 +337,274 @@ def _load_mesh(self):  # {{{
+                           self.endYearClimo)
+        if not os.path.exists(outputFileClimo):
+            self.logger.info('   Load data...')
+
+            climatologyFileName = self.mpasClimatologyTask.get_file_name(
+                season='ANN')
+            annualClimatology = xr.open_dataset(climatologyFileName)
+            annualClimatology = annualClimatology.isel(Time=0)
+
+            # rename some variables for convenience
+            annualClimatology = annualClimatology.rename(
+                {'timeMonthly_avg_mocStreamvalLatAndDepth': 'avgMocStreamfunGlobal',
+                 'timeMonthly_avg_mocStreamvalLatAndDepthRegion': 'avgMocStreamfunRegional'})
+
+            # Create dictionary for MOC climatology (NB: need this form
+            # in order to convert it to xarray dataset later in the script)
+            self.depth = refZMid
+            self.lat = {}
+            self.moc = {}
+            for region in self.regionNames:
+                self.logger.info('   Compute {} MOC...'.format(region))
+                if region == 'Global':
+                    mocTop = annualClimatology.avgMocStreamfunGlobal.values
+                else:
+                    # hard-wire region=0 (Atlantic) for now
+                    indRegion = 0
+                    mocTop = annualClimatology.avgMocStreamfunRegional[indRegion, :, :].values
+                # Store computed MOC to dictionary
+                self.lat[region] = binBoundaryMocStreamfunction
+                self.moc[region] = mocTop
+
+            # Save to file
+            self.logger.info('   Save global and regional MOC to file...')
+            ncFile = netCDF4.Dataset(outputFileClimo, mode='w')
+            # create dimensions
+            ncFile.createDimension('nz', len(refTopDepth))
+            for region in self.regionNames:
+                latBins = self.lat[region]
+                mocTop = self.moc[region]
+                ncFile.createDimension('nx{}'.format(region), len(latBins))
+                # create variables
+                x = ncFile.createVariable('lat{}'.format(region), 'f4',
+                                          ('nx{}'.format(region),))
+                x.description = 'latitude bins for MOC {}'\
+                                ' streamfunction'.format(region)
+                x.units = 'degrees (-90 to 90)'
+                y = ncFile.createVariable('moc{}'.format(region), 'f4',
+                                          ('nz', 'nx{}'.format(region)))
+                y.description = 'MOC {} streamfunction, annual'\
+                                ' climatology'.format(region)
+                y.units = 'Sv (10^6 m^3/s)'
+                # save variables
+                x[:] = latBins
+                y[:, :] = mocTop
+            depth = ncFile.createVariable('depth', 'f4', ('nz',))
+            depth.description = 'depth'
+            depth.units = 'meters'
+            depth[:] = self.depth
```

Comment:
@xylar: I think I took care of most of the changes needed here. 
Note the change on the line above `depth[:] = self.depth`, with respect to the `_compute_moc_climo_postprocess` respective line. I think we should change it in there too, but I'd like to double check with you first.

Reply:
I went ahead and made this change in `_compute_moc_climo_postprocess` as well.

Reply:
Yep, seems fine to me.  We might want to think about how this would change for a non-z-level grid in the future but that's a separate issue and there's already a GitHub issue for it (#120)